src/product.html

```diff
@@ -62,18 +62,18 @@
           </section>
         </header> <!-- Risky Business Ends here -->
         <section class="col col-2-3"> <!-- Left Column Starts Here -->
-          <section class="img-slider" ng-app="slider-module"> <!-- Air Drummers Starts here -->
+          <section class="img-slider"> <!-- Air Drummers Starts here -->
             <div class="img-main">
               <a href="#" id="prev"><div class = "arrow-case"></div><i class="fa fa-arrow-left"></i></a>
-              <img class= "img-active" src={{listingImages.results[0].url_570xN}}>
+              <img class= "img-active" ng-src={{listingImages.results[0].url_570xN}}>
               <a href="#" id="next"><div class = "arrow-case"></div><i class="fa fa-arrow-right"></i></a>
             </div>
             <ul class="img-list">
-              <li class="img-list-items img1"><img class="active" src={{listingImages.results[0].url_75x75}}></li>
-              <li class="img-list-items img2"><img src={{listingImages.results[1].url_75x75}}></li>
-              <li class="img-list-items img3"><img src={{listingImages.results[2].url_75x75}}></li>
-              <li class="img-list-items img4"><img src={{listingImages.results[3].url_75x75}}></li>
-              <li class="img-list-items img5"><img src={{listingImages.results[4].url_75x75}}></li>
+              <li class="img-list-items img1"><img class="active" ng-src={{listingImages.results[0].url_75x75}}></li>
+              <li class="img-list-items img2"><img ng-src={{listingImages.results[1].url_75x75}}></li>
+              <li class="img-list-items img3"><img ng-src={{listingImages.results[2].url_75x75}}></li>
+              <li class="img-list-items img4"><img ng-src={{listingImages.results[3].url_75x75}}></li>
+              <li class="img-list-items img5"><img ng-src={{listingImages.results[4].url_75x75}}></li>
```

Comment:
Yes I am thinking that, these codes were last Sunday's and was just targeting to make the images appear.